packages/paste-core/components/combobox/__tests__/index.spec.tsx

```diff
@@ -289,7 +292,14 @@ describe('Combobox', () => {
   describe('Accessibility', () => {
     it('Should have no accessibility violations', async () => {
       const {container} = render(<ComboboxMock />);
-      const results = await axe(container);
+      const results = await axe(container, {
+        rules: {
+          // Incorrectly thinks our markup is bad for a11y.
+          // False negative
```

Comment:
nit: If you wanted to dig out that axe-core github issue this relates to it might be useful. But not really important